themes/default/layouts/index.html

```diff
@@ -271,29 +267,47 @@ <h6>{{ $item.title }}</h6>
         </div>
     </section>
 
-    <!-- Benefits -->
-    <section class="container mx-auto my-12 lg:my-24 relative">
-        <h2 class="text-center">{{ .Params.benefits.title }}</h2>
-        <div class="benefits-section relative">
-            <div class="shape-background home-benefits-section">
-                <div class="shape-container">
-                    <div class="circle-left"></div>
-                    <div class="circle-right"></div>
+    <section class="container mx-auto my-16 text-center px-3">
+        <div class="card bg-violet-200 p-6 rounded-2xl">
+            <h2 class="rainbow-text">Doing more with less</h2>
+            <p class="text-lg max-w-3xl mx-auto">
+                You can use Pulumi's Automation API to create tooling that help you and your engineers manage <span class="font-bold">10x the amount of resources</span> versus
+                traditional tooling.
+            </p>
+
+            <div class="flex flex-wrap text-left">
+                <div class="w-full lg:w-1/3 p-6">
+                    <p>
+                        What if IaC was a library and not just a CLI? Pulumi Automation API is the industry's first and only fully programmable infrastructure as code technology.
```

Comment:
Nit, but "were" would be slightly better here. 

```suggestion
                        What if IaC were a library and not just a CLI? Pulumi Automation API is the industry's first and only fully programmable infrastructure as code technology.
```

Reply:
I generally use IaC as a singular noun.

Reply:
Oh, it's still singular -- just a different verb tense (the subjunctive): https://www.grammarly.com/blog/was-vs-were/